<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SleepReportWindowBasic</class>
 <widget class="QWidget" name="SleepReportWindowBasic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="SleepReportWindowBasicBk">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>272</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>480</width>
     <height>272</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 49, 49);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="homeSleepReportButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>24</y>
      <width>38</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
border-radius:3px;
background-color: rgb(70, 70, 73);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../rsc/resource.qrc">
      <normaloff>:/icons/home.png</normaloff>:/icons/home.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="moreSleepReportButton">
    <property name="geometry">
     <rect>
      <x>433</x>
      <y>24</y>
      <width>38</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
border-radius:3px;
background-color: rgb(70, 70, 73);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../rsc/resource.qrc">
      <normaloff>:/icons/next.png</normaloff>:/icons/next.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="sleepReportTitle">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>24</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#1788c3
</string>
    </property>
    <property name="text">
     <string>Sleep Report</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>391</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="lastRecordTime">
       <property name="styleSheet">
        <string notr="true">background-color:#3D3D3D;
border-radius:3px;
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="lastRecordTimeTitle">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>4</y>
          <width>151</width>
          <height>18</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#BDBDBD</string>
        </property>
        <property name="text">
         <string>Last Recorded Therapy</string>
        </property>
       </widget>
       <widget class="QLabel" name="lastRecordTimeText">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>361</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#2196F3</string>
        </property>
        <property name="text">
         <string>Wednesday,December 6</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="usageTime">
       <property name="styleSheet">
        <string notr="true">background-color:#3D3D3D;
border-radius:3px;
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="usageTimeTitle">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>4</y>
          <width>211</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#BDBDBD</string>
        </property>
        <property name="text">
         <string>Usage Hours</string>
        </property>
       </widget>
       <widget class="QLabel" name="usageTimeIcon">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>19</y>
          <width>28</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../rsc/resource.qrc">:/icons/usage_time.png</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="usageTimeStatus">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>0</y>
          <width>35</width>
          <height>47</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#2196F3;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="usageTimeUnit">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>30</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#2196F3</string>
        </property>
        <property name="text">
         <string>2.00-3.99</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="eventsPerHour">
       <property name="styleSheet">
        <string notr="true">background-color:#3D3D3D;
border-radius:3px;
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="eventsPerHourTitle">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>4</y>
          <width>201</width>
          <height>18</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#BDBDBD</string>
        </property>
        <property name="text">
         <string>Events ( A H I ) Per Hour</string>
        </property>
       </widget>
       <widget class="QLabel" name="evntsPerHourIcon">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>28</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../rsc/resource.qrc">:/icons/reminders.png</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="eventPerHourStatus">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>0</y>
          <width>35</width>
          <height>47</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>23</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#2196F3;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="eventsPerHourUnit">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>30</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#2196F3</string>
        </property>
        <property name="text">
         <string>10-19.99</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="maskFit">
       <property name="styleSheet">
        <string notr="true">background-color:#3D3D3D;
border-radius:3px;
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="maskFitTitle">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>4</y>
          <width>111</width>
          <height>18</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#BDBDBD</string>
        </property>
        <property name="text">
         <string>Mask Fit</string>
        </property>
       </widget>
       <widget class="QLabel" name="maskFitIcon">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>28</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../rsc/resource.qrc">:/icons/mask_fit.png</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="mskFitStatus">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>0</y>
          <width>35</width>
          <height>47</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../rsc/resource.qrc"/>
 </resources>
 <connections/>
</ui>
